.vscode/launch.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
I don't use VSCode so cannot really review / test this file. However, it seems like there are some run-specific arguments embedded into it. If we were to include it, I'd prefer a more generic version (maybe instead a doc describing how to generate this file)